lib/oelite/cookbook.py

```diff
@@ -851,3 +853,43 @@ def compute_recipe_build_priorities(self):
             c.remaining_tasks = len(c.tasks)
             del descendants[c]
         assert(len(descendants) == 0)
+
+    def create_world_recipes(self):
+        tmpdir = self.config.get("TMPDIR") or "tmp"
+        recipe_file = os.path.join(tmpdir, "world.oe")
```

Comment:
sure, but you do realize there's only one file involved? The plural `recipes` is because that file declares multiple recipe types, so it ends up defining multiple OEliteRecipes.